src/custom.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module '*.svg' {
+    export const ReactComponent: React.FunctionComponent<
+        React.SVGAttributes<SVGElement>
+    >;
+
+    const src: string;
+    export default src;
+}
```

Comment:
This is a pure copy/paste from https://github.com/DataBiosphere/duos-ui/pull/2512/files#diff-5f2ca1b1541dc3023f81543689da349e59b97c708462dd8da4640b399362edc7